apps/web/src/components/mapping/google-maps/change-map-style.tsx

```
@@ -0,0 +1,102 @@
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+} from "@/components/ui/popover";
+import { cn } from "@/lib/utils";
+import { api } from "@buzztrip/backend/api";
+import { Id } from "@buzztrip/backend/dataModel";
+import { MapTypeId } from "@buzztrip/backend/types";
+import { useMap } from "@vis.gl/react-google-maps";
+import { useMutation } from "convex/react";
+import { Map } from "lucide-react";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)